### PR TITLE
simplify code contributions by fully automating the dev setup with Gitpod

### PR DESCRIPTION
**Pull request by nisarhassan12**
This Pr simplifies code contributions by fully automating the dev setup with Gitpod(an online open-source VS Code like ide). With a single click, it will launch a workspace and automatically:

- clone the developerFolio repo.
- install the dependencies.
- start the dev server.

So that anyone interested in contributing or playing around with the code can start straight away without any friction.

You can give it a try on my fork of the repo via the link: 

https://gitpod.io/#https://github.com/nisarhassan12/developerFolio

This is how it looks:

image

### CODE REVIEW

**Comment by kartikcho** [comment]
While I really like the benefits of Gitpod, I think it still works without a config if you just prefix the GitHub URL with `gitpod.io/#`.

I'm not exactly for the decision of adding this but it isn't hurting the project in any way so I don't have a strong opinion.

cc: @saadpasta @naveen521kk

File: .gitpod.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+tasks:
+  - name: Dev Server
+    init: yarn install
+    command: yarn start
```
Comment (naveen521kk) [inline .gitpod.yml:4]: Using `npm` is a lot better. AFAIK. because `yarn.lock` is removed in #255

**Comment by saadpasta** [comment]
@nisarhassan12 That's a great idea and Thanks for checking out DeveloperFolio.I think we don't need this for now.

**Comment by nisarhassan12** [comment]
Ok. Thanks.

**Pull request closed without merging**